harness/resizableArrayBufferUtils.js

```diff
@@ -149,3 +149,10 @@ function TestIterationAndResize(iterable, expected, rab, resizeAfter, newByteLen
   }
   assert(resized, "TestIterationAndResize: resize condition should have been hit");
 }
+
+function MayNeedBigInt(ta, n) {
+  if (typeof n == 'number' && (ta instanceof BigInt64Array || ta instanceof BigUint64Array)) {
```

Comment:
Hm, I see some of the files did have the typeof check, but it seems to me that calling this function when `n` is not a number would be a test bug - I think I'd rather put an `assert.sameValue(typeof n, 'number')` at the start. (But please correct me if there's a reason not to do that.)

Reply:
Done (there was no reason to have that as a conditional).